watertap/costing/units/electroNP.py

```diff
@@ -104,7 +103,7 @@ def cost_electroNP_capital(blk, HRT, sizing_cost):
     blk.sizing_cost = pyo.Expression(expr=sizing_cost)
 
     flow_in = pyo.units.convert(
-        blk.unit_model.properties_in[0].flow_vol,
+        blk.unit_model.mixed_state[0].flow_vol,
```

Comment:
Just want to confirm with you, the `proerties_in` will be converted `mixed_state`, `properties_treated` will be converted to `treated_state`, and `properties_byproduct` will be converted to `byproduct_state`

Reply:
yes in order to use the IDAES separator

Reply:
We've done things like that in other models (e.g., RO).

Reply:
Yeah, I agree with Adam, if you could follow the conventions to rename the `mixed_state`, it would be really helpful.